naxsi_src/naxsi_runtime.c

```diff
@@ -809,7 +809,8 @@ ngx_int_t ngx_http_nx_log(ngx_http_request_ctx_t *ctx,
   const char 	*fmt_base = "ip=%.*s&server=%.*s&uri=%.*s&vers=%.*s&total_processed=%zu&total_blocked=%zu&config=%.*s";
   const char	*fmt_score = "&cscore%d=%.*s&score%d=%zu";
   const char	*fmt_rm = "&zone%d=%s&id%d=%d&var_name%d=%.*s";
-  const char    *fmt_config =  ctx->learning ? (ctx->drop ? "learning-drop" : "learning" ) :  (ctx->drop ? "drop" : (ctx->block ? "block" : ""));
+  const char    *fmt_config =  ctx-> log ? (ctx->learning ? (ctx->drop ? "log-learning-drop" : "log-learning" ) :  (ctx->drop ? "log-drop" : (ctx->block ? "log-block" : ""))) : \
```

Comment:
I think that ternary is big enough that a good ol' if/else would help readability ;)